packages/create-sitecore-jss/src/templates/nextjs/scripts/generate-component-factory-creator.ts

```diff
@@ -59,93 +54,20 @@ function watchComponentFactoryCreator() {
     return project.componentsPath;
   });
 
-  watchItems([componentRootPath, ...projectComponentsPaths], writeComponentFactoryCreator);
+  const componentFactoryCreatorWrapper = () => 
+  {
+    writeComponentFactoryCreator(config.componentRootPath, config.projectRootPath);
```

Comment:
Not sure I follow the use of `config` for `componentRootPath` and `projectRootPath`. Shouldn't it be using `constants` from '@sitecore-jss/sitecore-jss-dev-tools/nextjs' (same as bootstrap plugin)?

Reply:
That is a stray change from before that should be reverted. Thanks for noticing it!